### PR TITLE
creating hello world program

### PR DESCRIPTION
**Pull request by chloesun**
This change will allow us to print "hello world" to the console in a crazy sweet way using ruby.

Review : @ShopifyOnboarding/bcs-mentors

### CODE REVIEW

**Comment by JackTLi** [comment]
Hmm, looks like the "Hello world" part is missing, the file looks empty :/

**Comment by chloesun** [comment]
@JackTLi  a new push

File: chloe/helloworld.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+hello world
```
Comment (christinejordan) [inline chloe/helloworld.rb]: I think you might mean `puts Hello World!`.

**Comment by JackTLi** [comment]
👍

**Pull request merged**